newyork.html

```diff
@@ -522,105 +619,112 @@ <h2>NEWS</h2>
      </div>
 
 
+
      <div class="footer-box">
```

Comment:
## Final Changes in the `footer`
- [ ] The footer might need the `logo` to be added. The font size might also need to be improved. Enlarged and boldened in some instance as shown in the comparison below. 
- [ ] This part also has a sticky part at the bottom that has not been implemented. Is this a possible fix?

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|![footer](https://user-images.githubusercontent.com/22345961/108572927-768e8280-7324-11eb-9cd3-d83052155b56.png)|![footer-clone](https://user-images.githubusercontent.com/22345961/108572943-7abaa000-7324-11eb-96c7-53cc901e3efd.png)|